CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ## StreamChat
+### ✅ Added
+- Add `completion` to disconnect APIs [#2016](https://github.com/GetStream/stream-chat-swift/issues/2016)
```

Comment:
It looks like we actually removed this. Can we remove this changelog? 🤔

Reply:
Fixed